waku/v2/protocol/waku_store/waku_store_types.nim

```diff
@@ -52,9 +52,15 @@ type
     startTime*: float64 # used for time-window query
     endTime*: float64 # used for time-window query
 
+  HistoryResponseError* {.pure.} = enum
+    ## HistoryResponseError contains error message to inform  the querying node about the state of its request
+    OK = uint32(0)
```

Comment:
Why is this an error?

Reply:
I'd suggest making this `NONE` and the default. That way we're not mixing "Success" and "Error" codes (which should be avoided) and don't have to use it other than for error conditions.

Reply:
Sure, that makes sense. I will change its name to none!

Reply:
OK means no error, and I agree that NONE makes more sense! @oskarth